### ISSUE TITLE
Web tutorial step 9, firebase deploy can fail with authorization error

### ISSUE THREAD

**Issue by DanielSank** [firebase/friendlychat-web#74]
On step 9 of the web tutorial [1], after creating `database-rules.json` and modifying `firebase.json`, I try to deploy, and get the following error:

```
~/src/friendlychat/web-start$ firebase deploy

Error: Unable to authorize access to project friendlychat

Note: This version of the Firebase CLI is only compatible with projects upgraded
to the new Firebase Console. To access firebase.com apps, you will need to
use a previous version: npm install -g firebase-tools@^2.1

To access the Firebase Console, visit https://console.firebase.google.com/
```

From an SO post about a similar issue [2], I tried `firebase use --add`, even though I had done this about a day ago when I first started the tutorial. This solved the problem. Therefore I recommend mentioning, at some point, that `firebase use --add` may need to be re-invoked.

[1] https://codelabs.developers.google.com/codelabs/firebase-web/#8

[2] http://stackoverflow.com/questions/37381702/firebase-hosting-unable-to-authorize-access-to-project

**Comment by nicolasgarnier** [comment]
Hey @DanielSank 

Thanks for the report. It may be a bug in the Firebase CLI since `firebase use --add` should not need to be re-invoked. 

Can you confirmed you had already ran `firebase use --add` earlier on step 5 ? Is your project ID just `friendlychat` ?

We'll have a look into it!

**Comment by DanielSank** [comment]
> Can you confirmed you had already ran firebase use --add earlier on step 5?

I'm *pretty sure* I ran `firebase use --add` yesterday when I started setting all this stuff up. When I did it today and got the prompt about project alias I'm 100% sure I remember seeing that before, particularly because I have written notes about what alias I chose ;)

> Is your project ID just friendlychat ?

I'm not sure how to find the project ID, but "FriendlyChat" is what I see in the firebase console near the top right in the dropdown list of project names.

**Comment by nicolasgarnier** [comment]
You can find the ID of your project in the URL of the Firebase console:

firebase.corp.google.com/project/**&lt;project_id&gt;**/overview

I just want to make sure the Firebase Cli had the correct ID to begin with :)

**Comment by DanielSank** [comment]
Here's the URL of the console page

`.../project/friendlychat-4600e/database/data`

I guess the ID is `friendlychat-4600e`. Note, by the way, that I'm doing this on a Google corp account.

**Comment by nicolasgarnier** [comment]
Thanks so what's weird is that when yo had the CLI error the CLIE thought your app ID was simply `friendlychat` from the error line below:

```
Error: Unable to authorize access to project friendlychat
```

**Comment by nicolasgarnier** [comment]
@mbleigh Have you heard of similar issues in the CLI?

**Comment by teddyzetterlund** [comment]
Thought I'll chip in here, since I had the same error message and fix. Though, not unexpected when you think about it. I removed my local code folder for my Firebase projects, followed by cloning it anew from GitHub. Then I simply wrote `firebase deploy` by habit and the error message above threw me off a bit, which is why I found this thread.

So that could be a way to replicate it at least.

**Comment by Gsaico** [comment]
I have the same problem

**F:\ProyectosFirebase\friendlychat\web-start>npm  install firebase-tools**

npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
npm WARN prefer global firebase-tools@3.10.0 should be installed with -g
friendlychat-codelab@ F:\ProyectosFirebase\friendlychat\web-start
`-- firebase-tools@3.10.0

npm WARN friendlychat-codelab@ No repository field.
npm WARN friendlychat-codelab@ No license field.

**F:\ProyectosFirebase\friendlychat\web-start>firebase --version
3.10.0**

**F:\ProyectosFirebase\friendlychat\web-start>firebase login
Already logged in as saicos@gmail.com**

F:\ProyectosFirebase\friendlychat\web-start>firebase use --add
? Which project do you want to add? practicas-e412a
? What alias do you want to use for this project? (e.g. staging) friendlychatweb

Created alias friendlychatweb for practicas-e412a.
Now using alias friendlychatweb (practicas-e412a)

**F:\ProyectosFirebase\friendlychat\web-start>firebase serve**

Error: Unable to authorize access to project friendlychatweb

**F:\ProyectosFirebase\friendlychat\web-start>firebase serve**

Error: Unable to authorize access to project friendlychatweb

Having trouble? Try firebase serve --help

F:\ProyectosFirebase\friendlychat\web-start>

**Comment by nicolasgarnier** [comment]
OK this was an issue with firebase-tools 3.10.0 which got fixed in 3.10.1 where firebase tools was using the alias name instead of the project ID.

**Comment by rohit75** [comment]
run > firebase add (Inside Project folder)
? Which project do you want to add? <Specify your project id>
? What alias do you want to use for this project? (e.g. staging)<u can specify development>
You can change alias in future
here is the link- https://firebase.google.com/docs/cli/
